Formula/visp.rb

```diff
@@ -83,7 +83,20 @@ def install
                          "-DZBAR_LIBRARIES=#{Formula["zbar"].opt_lib}/libzbar.dylib",
                          "-DUSE_ZLIB=ON",
                          *std_cmake_args
+
+    # Replace generated references to OpenCV's Cellar path
+    opencv = Formula["opencv"]
+    opencv_references = [
+      "unix-install/vpConfig.h",
+      "unix-install/VISPConfig.cmake",
+      "VISPConfig.cmake",
+      "CMakeCache.txt",
+    ]
+    inreplace opencv_references, opencv.prefix.realpath, opencv.opt_prefix
     system "make", "install"
+
+    # Make sure software built against visp don't reference opencv's cellar path either
+    inreplace lib/"pkgconfig/visp.pc", opencv.prefix.realpath, opencv.opt_prefix
```

Comment:
This might error out now that I've done a bunch of `inreplace`s above, but let's see.